src/entities/trade.ts

```diff
@@ -39,6 +41,11 @@ export class Trade<TInput extends Currency, TOutput extends Currency, TTradeType
       outputAmount: CurrencyAmount<TOutput>
     }[]
     tradeType: TTradeType
+    mixedRoutes?: {
+      mixedRoute: MixedRouteSDK<TInput, TOutput>
+      inputAmount: CurrencyAmount<TInput>
+      outputAmount: CurrencyAmount<TOutput>
+    }[]
```

Comment:
I chose to add this as the last parameter and to make it optional for maximum backwards compatibility - usage of the `Trade` class across other repos would have to change if it was different.